content/en/docs/ibm/install-kubeflow.md

```diff
@@ -198,44 +210,100 @@ Run the following commands to set up and deploy Kubeflow.
       ```
       tar -xvf kfctl_{{% kf-latest-version %}}_<platform>.tar.gz
       ```
+1. (Optional) make kfctl binary easier to use. If you don’t add the binary to your path, you must use the full path to the kfctl binary each time you run it.
```

Comment:
```suggestion
1. Make kfctl binary easier to use (optional). If you don’t add the binary to your path, you must use the full path to the kfctl binary each time you run it.
```

Reply:
addressed in https://github.com/kubeflow/website/pull/2028/commits/9c471a4aecf999fdeb136ae19e68240ba9b290cb